code/modules/antagonists/eldritch_cult/knowledge/void_lore.dm

```diff
@@ -0,0 +1,199 @@
+	. = ..()
+	if(!ishuman(target) || !iscarbon(user))
+		return
+	var/mob/living/carbon/carbon_human = user
+	var/mob/living/carbon/human/human_target = target
+	var/datum/status_effect/eldritch/effect = human_target.has_status_effect(/datum/status_effect/eldritch/rust) || human_target.has_status_effect(/datum/status_effect/eldritch/ash) || human_target.has_status_effect(/datum/status_effect/eldritch/flesh) || human_target.has_status_effect(/datum/status_effect/eldritch/void)
+	if(!effect)
+		return
+	var/dir = angle2dir(dir2angle(get_dir(user,human_target))+180)
+	carbon_human.forceMove(get_step(human_target,dir))
+	var/obj/item/melee/sickly_blade/blade = carbon_human.get_active_held_item()
+	blade.melee_attack_chain(carbon_human,human_target)
+
+/datum/eldritch_knowledge/spell/voidpull
+	name = "Void Pull"
+	gain_text = "This entity calls itself the aristocrat, I'm close to ending what was started."
+	desc = "You gain an ability that let's you pull people around you closer to you."
+	cost = 1
+	spell_to_add = /obj/effect/proc_holder/spell/targeted/void_pull
+	next_knowledge = list(/datum/eldritch_knowledge/final/void_final,/datum/eldritch_knowledge/spell/blood_siphon,/datum/eldritch_knowledge/summon/rusty)
+	route = PATH_VOID
+
+/datum/eldritch_knowledge/final/void_final
+	name = "Waltz at the End of Time"
+	desc = "Bring 3 corpses onto the transmutation rune. After you finish the ritual you will automatically silence people around you and will summon a snow storm around you."
+	gain_text = "The world falls into darkness. I stand in an empty plane, small flakes of ice fall from the sky. Aristocrat stand before me, he motions to me. We will play a waltz to the whispers of dying reality, as the world is destroyed before our eyes."
+	cost = 3
+	required_atoms = list(/mob/living/carbon/human)
+	route = PATH_VOID
+	var/datum/looping_sound/void_loop/sound_loop
+	var/datum/weather/void_storm/storm
+
+/datum/eldritch_knowledge/final/void_final/on_finished_recipe(mob/living/user, list/atoms, loc)
+	var/mob/living/carbon/human/H = user
+	H.physiology.brute_mod *= 0.5
+	H.physiology.burn_mod *= 0.5
+	// COMMENTED FOR TESTMERGE
+	//H.client?.give_award(/datum/award/achievement/misc/void_ascension, H)
+	priority_announce("$^@&#*$^@(#&$(@&#^$&#^@# The nobleman of void [H.real_name] has arrived, step along the Waltz that ends worlds! $^@&#*$^@(#&$(@&#^$&#^@#","#$^@&#*$^@(#&$(@&#^$&#^@#", 'sound/ai/spanomalies.ogg')
+
+	sound_loop = new(list(user),TRUE,TRUE)
+	return ..()
+
+/datum/eldritch_knowledge/final/void_final/on_death()
+	if(sound_loop)
+		sound_loop.stop()
+	if(storm)
+		storm.wind_down()
+
+/datum/eldritch_knowledge/final/void_final/on_life(mob/user)
+	. = ..()
+	if(!finished)
+		return
+
+	for(var/mob/living/carbon/livies in spiral_range(7,user)-user)
+		if(IS_HERETIC_MONSTER(livies) || IS_HERETIC(livies))
+			return
+		livies.silent += 1
```

Comment:
you've tested to make sure that this properly silences people all of the time, right? I'm worried that the ticks that remove points of silent could get out of sync with these ticks that apply silent.

maybe use silent = max(2, silent+1) or something? idk, this might not even be a problem.

Reply:
it shouldnt